build-support/common.sh

```diff
@@ -61,3 +61,8 @@ function end_travis_section() {
 function fingerprint_data() {
   git hash-object -t blob --stdin
 }
+
+function git_merge_base() {
+  # This prints the tracking branch if set and otherwise falls back to local "master".
+  git rev-parse --symbolic-full-name --abbrev-ref HEAD@{upstream} 2>/dev/null || echo master
```

Comment:
Thanks for using full arg names.

This is nitty but think `echo 'master'` is preferred. Danny explained to me it's good to use quotes to completely disambiguate that `master` is not a function, but simply a string.

Reply:
Presumably disambiguate for humans - its completely unambiguous to bash IIUC since `echo` is the command and all further args until EOL or ; or ||, etc go to it; ie there would need to be a `master` (which 'master' now looks like - visual ambiguity!) or `$(master)`. Done.